README.md

```diff
@@ -944,6 +1019,58 @@ For the example above, this path will resolve to `/terraform-code/frontend-app/.
 what you want.
 
 
+#### get_parent_dir
```

Comment:
Done

I did not add a test for get_aws_account_id() since the expected result would be different and validating the expected result would require to implement the exact same function in the test.